gradle.properties

```diff
@@ -1 +1 @@
-version=0.6.2
+version=0.7.0-SNAPSHOT
```

Comment:
That probably needs its own PR

Reply:
should have been part of 0.6.2 release.

Reply:
https://github.com/hyperledger/besu-native/pull/99